### PR TITLE
Routes and cluster table

### PR DESCRIPTION
**Pull request by AlinaGoaga**

### CODE REVIEW

File: src/components/Dashboard/index.tsx [hunk 1]
```diff
@@ -13,7 +22,32 @@ const Dashboard: FC = () => {
 
   return (
     <div className="Dashboard">
-      Dashboard switch placeholder
+      <Switch>
```
Comment (AlinaGoaga) [inline src/components/Dashboard/index.tsx:25]: Hey! I think we'll need to look at what is happening with the token as I can't access the routes (same issue as the earlier refresh)

File: src/components/Clusters/Table/Row.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { Cluster } from "helix-js/dist/types/resources/clusters";
+import React, { FC } from "react";
+import { timestampToLocaleString } from "../../../utils";
+
+const ClusterTableRow: FC<{ cluster: Cluster }> = ({ cluster }) => {
```
Comment (AlinaGoaga) [inline src/components/Clusters/Table/Row.tsx:5]: I couldn't find info on the nodes in the cluster - one of the requirements was to show the number of managers and workers but it doesn't seem to be in there. Something to look into

File: src/components/Dashboard/index.tsx (hunk 1 above)
Comment (WebMaestroFr) [reply src/components/Dashboard/index.tsx:25]: Okay I'll look into this today :+1:

File: src/components/Dashboard/index.tsx [hunk 3]
```diff
@@ -13,7 +22,32 @@ const Dashboard: FC = () => {
...
+        <Route component={DashboardHome} exact={true} path={`/`} />
```
Comment (WebMaestroFr) [inline src/components/Dashboard/index.tsx:26]: `path={\`/\`}` is a very expensive alternative to `path="/"`.

Comment (AlinaGoaga) [reply src/components/Dashboard/index.tsx:26]: Fair enough - will update before the merge

**Pull request merged**